test/IntegrationTests/ModuleTests.cs

```diff
@@ -56,6 +56,7 @@ public async Task Minimal()
         SetEnvironmentVariable(ConfigurationKeys.Metrics.Exporter, Constants.ConfigurationValues.None);
         SetEnvironmentVariable(ConfigurationKeys.Traces.ConsoleExporterEnabled, bool.FalseString);
         SetEnvironmentVariable(ConfigurationKeys.Metrics.ConsoleExporterEnabled, bool.FalseString);
+        SetEnvironmentVariable(ConfigurationKeys.Traces.OpenTracingEnabled, bool.FalseString);
```

Comment:
As False is default, this probably can be removed.

Reply:
This was explicitly set to collect all vars that could load something. But can be removed if you see no additional value.

Reply:
I like the idea. We can keep it here.